core/win32/callback.c

```diff
@@ -3774,6 +3776,32 @@ intercept_nt_continue(CONTEXT *cxt, int flag)
         LOG(THREAD, LOG_ASYNCH, 3, "target context:\n");
         DOLOG(3, LOG_ASYNCH, { dump_context_info(cxt, THREAD, true); });
 
+        /* Updates debug register values.
+         * FIXME should check dr7 upper bits, and maybe dr6
```

Comment:
My code just checks the lower bits of Dr7, the upper bits should be checked as well